package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "//1": "describes your app and its dependencies",
```

Comment:
Sorry! I should've put my comments:

- https://github.com/githubsatelliteworkshops/apps-with-graphql/pull/3#discussion_r420245931
- https://github.com/githubsatelliteworkshops/apps-with-graphql/pull/3#discussion_r420246474
- https://github.com/githubsatelliteworkshops/apps-with-graphql/pull/3#discussion_r420246799

Here instead